CHANGELOG.md

```diff
@@ -4,20 +4,21 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 ### Added
-- 
+-
 
 ### Changed
-- 
+-
 
 ### Removed
-- 
+-
 
 ## 3.6.1 - 2016-08-12
 
 ### Changed
 - **cf-icons:** Standardized the usage doc.
 - **cf-pagination:** Standardized the usage doc.
 - **cf-forms:** Standardized the usage doc.
+- **cf-tables:** [MAJOR] Atomized cf-tables.
```

Comment:
Move to unreleased. 

Also this made me notice that this PR should have been opened against `prerelease-ver-4`  and  not canary. We're going to do all the atomic work at once. There should be other changes in that branch.